src/core/fonts.js

```diff
@@ -2967,6 +2971,7 @@ class Font {
       )
     ) {
       glyph = new Glyph(
+        charcode,
```

Comment:
Please note that you still need to insert `charCode` before line 2963 above, otherwise this won't work as intended.

Reply:
🤦